### ISSUE TITLE
WSL and Dev Containers extension using different style setting prefixes

### ISSUE THREAD

**Issue by gregvanl** [microsoft/vscode-remote-release#7268]
- VSCode Version: 1.72
- Local OS Version: Windows 11

The WSL and Dev Containers extensions use different conventions for prefixing their settings.

In the Settings editor:
WSL settings have the prefix "Remote.WSL:"
Dev Containers settings have the prefix "Dev > Containers:"

![image](https://user-images.githubusercontent.com/12818376/193160047-71ae2940-b5ea-4030-add6-48ff5a44e35a.png)

In settings.json these are:
WSL - "remote.WSL.*"
Dev Containers - "dev.containers"

Not sure if the "Remote." prefix is needed for WSL to avoid settings name collision with other WSL extensions?

The implied heirarchy for the "Dev > Containers" settings seems odd. Should this just be "Dev Containers:"?

**Comment by aeschli** [comment]
Yup, I'm planning to change to wsl setting names too. The prefix will be wsl.

The arrow is something the Settings UI does when rendering the segments. To avoid it we would have to call the setting `devContainers` or `dev-containers`.
